stack/dashboard/base/builder.sh

```diff
@@ -41,6 +42,18 @@ if [ "${future}" = "yes" ];then
 fi
 wazuh_minor=$(echo ${version} | cut -c1-3)
 
+# Obtain Wazuh plugin URL
+if [ "${repository}" ];then
+    valid_url='(https?|ftp|file)://[-[:alnum:]\+&@#/%?=~_|!:,.;]*[-[:alnum:]\+&@#/%=~_|]'
```

Comment:
We should specify some type of control over the URL that is specified, it may be the case that a URL like the following is specified:

```
https://packages-dev-fail.wazuh.com/pre-release/ui/dashboard/wazuh-1-1.zip
```

This URL will be accepted and will be used to download and install the Wazuh app during the construction of the package, which would end up failing, which could be avoided by checking the code that returns a curl beforehand.

If the URL is not valid, it should show a message and end the execution with an error status instead of using a default Wazuh URL.